opencti-platform/opencti-front/src/private/components/data/connectors/WorkDetail.tsx

```diff
@@ -0,0 +1,87 @@
+import Paper from '@mui/material/Paper';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import LinearProgress from '@mui/material/LinearProgress';
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import TaskStatus from '../../../../components/TaskStatus';
+import { useFormatter } from '../../../../components/i18n';
+
+const useStyles = makeStyles(() => ({
```

Comment:
the use of makeStyles is deprecated, can you put the style in react tag instead ?